### ISSUE TITLE
Wrong Import of SVG

### ISSUE THREAD

**Issue by hansnolte** [MaurycyLiebner/enve#181]
I made an SVG with Affinty Designer and import it into enve (7_1_20).
Wrong order of the Layers, no section Layers and no transfer of group names.
The SVG:
[Typ.zip](https://github.com/MaurycyLiebner/enve/files/4882979/Typ.zip)

Various other programs have no problems with the Layers and group names.

This is how it looks in enve.
![enve](https://user-images.githubusercontent.com/7922627/86730205-90e24d00-c02e-11ea-988e-8f81aefd248d.jpg)

Affinity Designer:
![affinity-designer](https://user-images.githubusercontent.com/7922627/86730250-9b044b80-c02e-11ea-951d-5c26f1e92042.jpg)

Inkscape:
![inkscape](https://user-images.githubusercontent.com/7922627/86730286-a35c8680-c02e-11ea-9571-8dc75f5a9c89.jpg)

Gravit:
![gravit](https://user-images.githubusercontent.com/7922627/86730317-ad7e8500-c02e-11ea-9124-722243dc2b67.jpg)

And even in Firefox it is displayed correctly.
![firefox](https://user-images.githubusercontent.com/7922627/86730387-c38c4580-c02e-11ea-80d8-eaa94bcc8b48.jpg)

**Comment by AlexKiryanov** [comment]
Hello! I rebuilt your file in enve!
The problem was in mask. For some reason, when importing this parameter is not read. (Yes, this is some kind of problem ...)
All layers and objects are there. I just duplicated some of the layers and made them masks (eyes and mouth). File by link. https://drive.google.com/file/d/1K154SZ_5MQIs-UyNrkG-w3nJGARKgZEl/view?usp=sharing
![Untitled-1](https://user-images.githubusercontent.com/65244629/87015250-d4c38680-c1d5-11ea-918c-e4f473261e60.jpg)

**Comment by hansnolte** [comment]
Hello Alex,

that's nice of you, but it doesn't solve the problem.
My concern was that the import was not working properly.
It would be quite annoying if you had to revise every imported SVG to make it usable.
And as my screenshots have shown, the other programs can import the SVG without errors.

Many greetings
Hans

**Comment by AlexKiryanov** [comment]
I agree! I just decided to supplement your post. And indicate a possible problem with masks.
